mono/metadata/object-internals.h

```diff
@@ -1811,13 +1807,10 @@ mono_class_is_reflection_method_or_constructor (MonoClass *klass);
 MonoObject *
 mono_get_object_from_blob (MonoDomain *domain, MonoType *type, const char *blob, MonoError *error);
 
-MonoObjectHandle
-mono_class_get_ref_info (MonoClass *klass);
-
 gboolean
 mono_class_has_ref_info (MonoClass *klass);
 
-MonoObject*
+MonoReflectionTypeBuilder*
```

Comment:
Please don't do this.  The naming convention is `_raw` is only for functions that return a raw pointer to managed memory when a function without a suffix exists that returns a coop handle.  Just use `mono_class_get_ref_info` with a `MONO_HANDLE_CAST` and delete `mono_class_get_ref_info_raw` when all of its callers are gone.